app/src/main/java/nl/tudelft/bbw/blockchain/Acquaintance.java

```diff
@@ -15,14 +16,14 @@
     /**
      * Properties of a Acquaintance
      */
-    private ArrayList<Chain> multichain;
+    private List<List<Block>> multichain;
```

Comment:
As @ymte stated and myself: I think that the map is not the ideal candidate in this situation because our multichain is used to just carry data. You don't really have to iterate through it or select from it.